extensions/toggle-menu-bar/package.json

```diff
@@ -11,9 +11,9 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "toggle",
+      "name": "toggleCommand",
```

Comment:
Can we keep the old name since users might have set a hotkey/alias here, so in order to not unset that 🙂

Reply:
No problem!